notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Is the cleaning up section missing? Should we have some cleanup steps here, e.g. undeploying and deleting resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Ugh! I had one, but it must've somehow gotten deleted. Added back.